marketplace-and-integrations/integrations/algolia.md

```diff
@@ -54,6 +55,13 @@ The following configuration is required for working with the Algolia API:
 ```
 {% endcode %}
 
+Algolia comes with a set of predefined API keys:
+
+* Search-Only API key - a public API key used on the frontend for performing search queries.
+* Admin API key - used on the backend for create, update or delete operations on the indices. 
```

Comment:
Would it make sense to put these into a table?
I can do it once we merge the PR.

Reply:
Yes, sounds good. Thanks